src/inmanta/plugins.py

```diff
@@ -544,3 +585,16 @@ def wrapper(self, *args: object, **kwargs: object) -> Any:
     elif function is not None:
         fnc = curry_name(commands=commands, emits_statements=emits_statements, allow_unknown=allow_unknown)
         return fnc(function)
+
+
+@stable_api
+def deprecated(
+    function: Optional[Callable] = None, *, replaced_by: Optional[str] = None, **kwargs: Dict[str, object]
+) -> Callable:  # noqa: H801
```

Comment:
I don't find this violation code anywhere, what does it mean and why do we need it?

Reply:
actually it is present on the plugin decorator and I didn't remove it when I copy/pasted its definition. I'll remove it at both places and see if something complains

Reply:
I see, it could be an old violation that no longer exists.